assets/css/data.css

```
@@ -0,0 +1,139 @@
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+}
+
+.download-list {
+  padding: 0;
+  margin: 20px 0;
+}
+
+.download-item {
+  background-color: #ffffff;
+  border: 1px solid #e0e0e0;
+  border-radius: 8px;
+  padding: 20px;
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+.download-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+select {
+  border: 1px solid #ddd;
+  font-size: 14px;
+}
+
+.toggle-button, .download-link {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  display: inline-flex;
+  padding: 10px;
+}
+
+.toggle-button:hover, .toggle-button:focus, 
+.download-link:hover, .download-link:focus {
+  outline: 3px solid #005fcc;
+}
+
+.file-name {
+  font-style: italic;
+  display: inline-block;
+  text-align: right;
+  width: 100%;
+  color: grey;
+}
+
+.file-header {
+  display: flex;
+  align-items: center;
+}
+
+.file-preview-header {
+  color: grey;
+}
+
+.file-preview {
+  width: calc(100% - 20px);
+  max-width: calc(100% - 20px);
+  border-collapse: collapse;
+  margin: 20px;
+  margin-bottom: 8px;
+  margin-right: 0px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+  position: relative;
+  overflow: hidden;
+  border-radius: 10px;
+  background-color: #ffffff;
+  border: 2px solid #cc79a7;
+  font-size: 12px;
+}
+
+.file-preview th, .file-preview td {
+  padding: 8px;
+  text-align: left;
+  border: 1px solid #cc79a7;
+}
+
+.file-preview th {
+  color: #333333;
+  font-weight: bold;
+}
+
+.file-preview tbody tr:nth-child(odd) {
+  background-color: #f9f9f9;
+}
+
+.file-preview tr:hover > * {
+  background-color: #f1f1f1;
+}
+
+.file-preview td {
+  color: #555555;
```

I think we need to use a high contrast color since it is not so readable with a grey background. Maybe just black?

```suggestion
  color: black;
```